docs/getting-started/server/self-hosted/index.mdx

```diff
@@ -150,14 +118,21 @@ pwsh migrate.ps1 -selfhost
 This will create a new database called `vault_dev_self_host` and/or run unknown migrations against
 it.
 
-To keep your self-hosted database up to date, calling the script with the `-selfhost` argument in
-the future will execute new migrations against `vault_dev_self_host`.
+You must call `migrate.ps1` with the `-selfhost` option in the future to keep your self-host
+database up-to-date with any migrations.
 
 ### Define Installation Id and Key for Your Cloud Database
 
+:::note
+
+These instructions should be moved into a script in the `dev/` directory - please submit a PR if you
```

Comment:
Is this even something we want to do?

Seems like the thinking here is that everything necessary to setup local environments should be in the code repo, but that's by definition not what we're doing -- instructions are here.

I don't see a difference between simple, small scripts to setup and the rest of our descriptions here.

Reply:
I think that we should provide scripts as actual scripts that can be executed, rather than the user having to create it ad-hoc.

I also think "run this script to generate and seed your installation keys" are better instructions than "you need to generate some installation keys, you can do it this way, or this other way, and if you like here's a script that might help". We should have a single, preferred way of doing an action.

But all of that said - it's not the main focus of this PR at all, so I'll just remove this callout.